src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackFrame.cs

```diff
@@ -66,10 +66,10 @@ public StackFrame()
         /// <summary>
         /// Constructs a StackFrame corresponding to the active stack frame.
         /// </summary>
-        public StackFrame(bool needFileInfo)
+        public StackFrame(bool fNeedFileInfo)
```

Comment:
This is another potentially undesirable change, however preserves compat.  cc @tommcdon

Reply:
@mikem8361